src/templates/static/affiliate_ib_landing/type_of_partnership.jsx

```diff
@@ -160,72 +160,77 @@ const TypesOfPartnership = () => (
                                     tbody: [
                                         [
                                             { text: it.L('Crash 1000 Index') },
-                                            { text: it.L('0.50') },
+                                            { text: it.L('0.25') },
```

Comment:
@negar-binary what is the reason we translated numbers in the first place? 😄

Reply:
@akmal-binary the separator in different languages is different. some use comma, some use space

Reply:
Oh wow, good to know, I thought some countries actually use different number system 😄